notebooks/book1/10/iris_logreg.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>Why plot is not showing here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>